### ISSUE TITLE
[JS, Python] Support `apiUrl` configuration option

### ISSUE THREAD

**Issue by ewanharris** [openfga/sdk-generator#298]
Currently only the .NET, Go, and Java SDKs support the `apiUrl` option that replaces the combination of `apiScheme` and `apiHost`.

We should also extend this to JS and Python, ensuring that we document using this instead of scheme/host and deprecate (?) scheme/host

[ ] JS
[ ] Python

**Comment by ewanharris** [comment]
cc @rhamzeh just incase there was a reason we didn't do these initially

**Comment by rhamzeh** [comment]
Yes, we're migrating all the SDKs to use apiUrl and deprecate the previous two!

JS and Python are the only two left
